doc/ddu.txt

```diff
@@ -1373,7 +1378,13 @@ CREATE SOURCE                                              *ddu-create-source*
 
 To create source, you should read other sources implementation.
 
-The sources must put under "denops/@ddu-sources/*.ts".
+The sources must be placed in one of the following structures, where {name} is
+the source name:
+
+	denops/@ddu-sources/{name}.ts
+	denops/@ddu-sources/{name}/main.ts
+
+NOTE: If both structures exist, "{name}.ts" is preferred.
```

Comment:
There is no description in the documentation that the "deno.json[c]" and "import_map.json[c]" files are also loaded.